desktop/core/ext-py3/dnspython-1.16.0/dns/query.py

```diff
@@ -257,22 +283,45 @@ def receive_udp(sock, destination, expiration=None,
     while 1:
```

Comment:
nit: logic is same (1 vs. True) https://github.com/rthalley/dnspython/blob/f66e25b5f549acf66d1fb6ead13eb3cff7d09af3/dns/query.py#L625